projects/sunbird-video-player/src/lib/sunbird-video-player.component.ts

```diff
@@ -227,6 +234,13 @@ export class SunbirdVideoPlayerComponent implements OnInit, AfterViewInit, OnDes
     this.QumlPlayerConfig.metadata['showEndPage'] = 'No';
     this.currentInterceptionTime = time;
     this.currentInterceptionUIId = identifier;
+    if (document.fullscreenElement) {
```

Comment:
Question set can also be played in full screen, i want to understand the reason for this changes

Reply:
For video fullscreen we are using video.js fullscreen so out quml player doesn't recognize the fullscreen and currently the Quml-player is not working thats why we are going with this approach.
In mobile application we launch the Interactive Video in landscape mode so even though video is in full screen or normal screen it doesnt affect the UI.